### ISSUE TITLE
Feature Request: Z-Lift on Top/First layers ONLY

### ISSUE THREAD

**Issue by Alonso3D** [supermerill/SuperSlicer#395]
### Version
Future versions

_Is this a new feature request?_
Yes!

Check box(es) to activate Z-Hop on first and top layers only. For top layers it would avoid leaving scars and save any extra wear and tear on the printer through out the rest of the print job. 

Perhaps also include a parameter for how many layers to activate Z-hop at the start/end of the objects.

For example, activate Z-hop only the first 2 layers of the print job and activate Z-hop on the LAST 3 layers of the object.
Basically we want it to do the opposite of what "Not on top" check box does .

**Comment by robthide37** [comment]
isn't this already available with the only lift z above and only lift z below paramter

**Comment by supermerill** [comment]
"top" is an extrusion type, it can happen at every layer.

**Comment by Kirinkunhime** [comment]
I +1 this feature request - having Z-Hop only on some layers would allow for some interesting options like (limited) multi-color prints without the need of filament change or an MMU.

**Comment by Matts-Hub** [comment]
My apologies for the duplicate, +1 for this feature request!
